packages/components/src/radio-group/stories/index.story.tsx

```diff
@@ -1,32 +1,41 @@
 /**
- * WordPress dependencies
+ * External dependencies
  */
-import { useState } from '@wordpress/element';
+import type { Meta, StoryFn } from '@storybook/react';
 
 /**
  * Internal dependencies
  */
-import { Radio, RadioGroup } from '../../radio-group-ariakit/';
+import { RadioGroup } from '..';
+import { Radio } from '../radio';
+import type { RadioProps } from '../types';
+
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
 
-export default {
-	title: 'Components (Deprecated)/RadioGroup',
+const meta: Meta< typeof RadioGroup > = {
 	component: RadioGroup,
-	subcomponents: { Radio },
+	title: 'Components (Deprecated)/RadioGroup (Ariakit)',
+	argTypes: {
+		// 	value: { control: { type: 'number', min: 0, max: 100, step: 1 } },
+		onChange: { action: 'onChange' },
```

Comment:
Instead of specifying action for each specific prop, we usually use RegEx ([example](https://github.com/WordPress/gutenberg/blob/f67c38677442518e65e338849eaa3fc435773a5c/packages/components/src/angle-picker-control/stories/index.story.tsx#L24))

Reply:
Fixed with https://github.com/WordPress/gutenberg/pull/55580/commits/1a9147860446ad23cd85ac37fb85237a3162afa4